collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+update_every = 10
+disabled_by_default = True
+
+DEFAULT_STATUS_MAP = {'CLEAR': 0, 'WARNING': 1, 'CRITICAL': 2}
+
+ORDER = [
+    'alarms',
+]
+
+CHARTS = {
+    'alarms': {
+        'options': ['alarms', 'Alarms', 'status', 'alarms', 'alarms.status', 'line'],
+        'lines': []
+    }
+}
+
+
+class Service(UrlService):
+    def __init__(self, configuration=None, name=None):
+        UrlService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+        self.status_map = self.configuration.get('status_map', DEFAULT_STATUS_MAP)
```

Comment:
```suggestion
        self.status_map = self.configuration.get('status_map', DEFAULT_STATUS_MAP)
        self.url = self.configuration.get('url', 'http://127.0.0.1:19999/api/v1/alarms?all')
```

i suggest to add default `url` so we not depend on the config file (would work w/o it)

https://github.com/netdata/netdata/blob/2c1d3e3a2f80cb9691edc8790398e050f9c44d7a/collectors/python.d.plugin/apache/apache.chart.py#L98